controllers/flowlogspipeline/flp_objects.go

```diff
@@ -245,46 +308,22 @@ func (b *builder) configMap() (*corev1.ConfigMap, string) {
 			{"name": "enrich",
 				"follows": "decode",
 			},
-			{"name": "encode",
-				"follows": "enrich",
-			},
 			{"name": "loki",
-				"follows": "encode",
+				"follows": "enrich",
```

Comment:
we don't need to have `encode` between `enrich` and `loki` ?

Reply:
It is empty, and not in use at all right now (we do not have any network metrics, so no enrichment or encoding) but I wanted to add those empty steps for completeness of the pipeline. This will change over time once we start to add metrics to the pipeline.